app/assets/stylesheets/vendor/SyntaxHighlighter/SyntaxHighlighter.scss

```diff
@@ -1,5 +1,4 @@
-@import 'mixins';
-@import 'constants';
+@import '../../common/constants';
```

Comment:
@arkon yeah this is kind of annoying. If you have time, can you look into the Hound config for CSS and change this one?